pkg/azurediskplugin/main.go

```diff
@@ -80,44 +97,138 @@ func main() {
 		klog.Warning("nodeid is empty")
 	}
 
+	getDriverConfig()
```

Comment:
Since the order of these function calls matters, i.e. `getDriverConfig` should be called before `handle`, we should move  `getDriverConfig` to `handle`. Same is true for `exportMetrics`.

Reply:
Also, see comment https://github.com/kubernetes-sigs/azuredisk-csi-driver/pull/1380/files#r921449306 for how make the order of these function calls explicit.